README.md

```diff
@@ -1,54 +1,59 @@
 Created by Jake Johnson and Preston Windfeldt
```

Comment:
There is a merge conflict because I added a couple things to readme for the ubuntu install